cibuildwheel/linux.py

```diff
@@ -191,13 +197,14 @@ def build(project_dir, output_dir, test_command, before_test, test_requires, tes
             ) done
         '''.format(
             pybin_paths=' '.join(c.path + '/bin' for c in platform_configs),
+            package_dir=container_package_dir,
             test_requires=' '.join(test_requires),
             test_extras=test_extras,
             test_command=shlex.quote(
-                prepare_command(test_command, project='/project') if test_command else ''
+                prepare_command(test_command, project=container_project_dir, package=container_package_dir) if test_command else ''
```

Comment:
Why the useless/constant `container_project_dir` variable?

Reply:
It is also used in various places in this file, so I thought it would be better to store it in one place.